tests/Resources/RHOSi.resource

```diff
@@ -42,6 +42,12 @@ RHOSi Setup
     Suite Availability Setup    ${RHODS_PROMETHEUS_URL}    ${RHODS_PROMETHEUS_TOKEN}
     # TO DO: oc login
 
+Get Observatorium Token
```

Comment:
to protect a kw you don't need to put the entire kw here. Follow the instructions here https://docs.google.com/document/d/1-88q364w4w8X9jpvIqY5UgYAUl6oOyM-neAOyq02vz4/edit#heading=h.ek10lx5xvvh1

You just need to add the kw name into the @PROTECTED_KEYWORDS list in the RHOSi.resource.

Reply:
added in @{PROTECTED_KEYWORDS}=